tests/test_qci_menu_visible_QCI92.py

```diff
@@ -0,0 +1,11 @@
+# https://polarion.engineering.redhat.com/polarion/redirect/project/QuickstartCloudInstaller/workitem?id=QCI-92
+
+# QuickStart Cloud Installer plugin menu is visible and clickable
```

Comment:
It says that it tests that it is clickable but the test does not click it?   Is this an oversight?